build.gradle.kts

```diff
@@ -125,6 +118,14 @@ val hivemqOpenApi: Configuration by configurations.creating {
     }
 }
 
+val hivemqOpenApiUpdater: Configuration by configurations.creating {
```

Comment:
The naming is weird, this is a configuration not an updater (you can not call update on it and then it performs an update).
Maybe we call it hivemqOpenApiFromProject